Dockerfile

```diff
@@ -1,5 +1,9 @@
+ARG BUILD_DATE
+ARG REVISION
+ARG TARGETPLATFORM
+
 FROM --platform=$BUILDPLATFORM tonistiigi/xx AS xx
-FROM --platform=$BUILDPLATFORM rust:alpine AS build
+FROM --platform=$BUILDPLATFORM rust:alpine3.19 AS build
```

Comment:
Why did you choose this particular tag granularity instead of `alpine` (like before) or a digest (for reproducability)? Is the aim to prevent breakage by non-patch updates? If so, why is the Rust version not specified here?

Reply:
Simply because I'm not aiming for reproducibility here. If I really want to aim for reproducibility, I would build the image with Nix.

Reply:
I'd still be more comfortable if we included the Rust version in the tag. Our closed-source Dockerfiles start with a `RUST_VERSION` arg at the top which is then interpolated in the `FROM` statement:

```Dockerfile
ARG RUST_VERSION=1.74.0
# ...
FROM rust:${RUST_VERSION}-slim-bookworm AS build-bundler
```

If you decide to add this, you should also add a comment, both in the Dockerfile and Cargo.toml that advises to keep this in sync with `workspace.package.rust-version`.

Reply:
Note that `workspace.package.rust-version` is the _minimum_ supported Rust version, not the one we mainly use.